config/manifests/bases/special-resource-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,57 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - GPI
+  links:
+  - name: README
+    url: https://github.com/openshift-psap/special-resource-operator
+  maintainers:
+  - email: zkaiser@redhat.com
+    name: Zvonko Kaiser
+  maturity: alpha
+  provider:
+    name: Red Hat
+    url: redhat.com
+  version: 0.0.0
```

Comment:
We need to set here something right?

Reply:
The version will be overwritten by the VERSION in the makefile when we `make bundle`